src/Colourista/Pure.hs

```diff
@@ -0,0 +1,74 @@
+{- |
+Copyright: (c) 2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+This module introduces helpful pure codes to customise the terminal output view.
+-}
+
+module Colourista.Pure
+    ( -- * Colour
+      red
+    , green
+    , blue
+    , yellow
+    , black
+    , white
+    , magenta
+    , cyan
+
+      -- * Emphasis
+    , bold
+    , italic
+
+      -- * Reset
+    , reset
+    ) where
+
+import System.Console.ANSI (Color (..), ColorIntensity (Vivid), ConsoleIntensity (BoldIntensity),
+                            ConsoleLayer (Foreground), SGR (..), setSGRCode)
+
+
+-- | Code to apply 'Red' colouring for the terminal output.
+red :: String
+red = setSGRCode [SetColor Foreground Vivid Red]
```

Comment:
I'm working on IO functions for this. So, what do you think, should our functions also work with `String` for compatibility with the library? Or should we wrap everything in `Text` or `ByteString`?

Reply:
I would love to have it in Text I guess, but I am not sure how compatible it will make everything. 

I see that best would be to have the returned type `IsString` so it could be compatible with everything. What are your thoughts?

Reply:
@vrom911 Having those functions polymorphic is an awesome idea! They can have `IsString` type indeed and we can add `SPECIALIZE` pragmas for `String`, `Text` and `ByteString` to have efficient versions without overhead for users!